frontend/src/pages/projects/components/DeleteModal.tsx

```diff
@@ -73,6 +73,11 @@ const DeleteModal: React.FC<DeleteModalProps> = ({
             aria-label="Delete modal input"
             value={value}
             onChange={(newValue) => setValue(newValue)}
+            onKeyDown={(event) => {
+              if (event.key === 'Enter' && value === deleteName) {
```

Comment:
```suggestion
              if (event.key === 'Enter' && value === deleteName && !deleting) {
```

You may also want to add this check to prevent multiple DELETE api calls. We do this check on the button so that the user cannot fire off a calls for the one resource. In theory you could spam enter a bunch of times and produce the same effect.

Reply:
This is a good suggestion to stop multiple submits to delete 👍